<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumTranslator</class>
 <widget class="QMainWindow" name="NumTranslator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>429</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>386</width>
    <height>429</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>386</width>
    <height>429</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Перевод чисел</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Test-case.ico</normaloff>Test-case.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>167</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="from_ss">
     <item>
      <widget class="QLabel" name="text_from">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Из какой с.с?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ss16">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>16</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ss10">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ss8">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ss4">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ss2">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>167</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="In_ss">
     <item>
      <widget class="QLabel" name="text_in">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>В какую с.с?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="iss16">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>16</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="iss10">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="iss8">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="iss4">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="iss2">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="create_test_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>СОЗДАТЬ</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>361</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="extra_set">
     <item row="1" column="0" colspan="2">
      <widget class="QRadioButton" name="fractional_num">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Дробные числа</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="Count_text">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Количество чисел:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="count_number"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
